File: src/main/java/mSearch/filmeSuchen/ListenerFilmeLaden.java
package mSearch.filmeSuchen;

import java.util.EventListener;

public class ListenerFilmeLaden implements EventListener {
    public void start(ListenerFilmeLadenEvent e) {
    }

    public void progress(ListenerFilmeLadenEvent e) {
    }

    public void fertig(ListenerFilmeLadenEvent e) {
    }

    public void fertigOnlyOne(ListenerFilmeLadenEvent e) {
        // dient dem Melden des ersten Mal Laden der Filmliste beim ProgStart
    }
}

File: src/main/kotlin/mediathek/filmeSuchen/ListenerFilmeLaden.kt
package mediathek.filmeSuchen

import java.util.EventListener

open class ListenerFilmeLaden : EventListener {
    open fun start(event: ListenerFilmeLadenEvent) = Unit

    open fun progress(event: ListenerFilmeLadenEvent) = Unit

    open fun fertig(event: ListenerFilmeLadenEvent) = Unit

    // dient dem Melden des ersten Mal Laden der Filmliste beim ProgStart
    open fun fertigOnlyOne(event: ListenerFilmeLadenEvent) = Unit
}
